.gitignore

```diff
@@ -110,3 +110,5 @@ venv.bak/
 ___*
 
 **cache/**
+cache/**
+testing/examples/cache/**
```

Comment:
Could the cache be always cleared with a try/finally in the tests?

Reply:
We could use a test class that would have the same setup and teardown for each tests.